src/api-v1.js

```diff
@@ -779,51 +735,3 @@ api.declare({
     uptime: process.uptime(),
   });
 });
-
-api.declare({
-  method: 'get',
-  route: '/backend-status',
```

Comment:
Please do not delete this.  It is supposed to be used to get information into status.taskcluster.net.  If that's not where the data for the aws-provisioner graph is coming from, then the graph is measuring the wrong thing.